modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -33,6 +33,8 @@ A user requires the following privileges to install an {product-title} cluster:
 ** *Assign virtual machine to resource pool*
 * Profile-driven storage
 ** All privileges
+* Tasks
+** All privileges
```

Comment:
We should also document that:

vCenter user configured in Openshift must have Read-Only permission to vCenter, Datacenter, Datastore Cluster, Datastore Storage Folder, Cluster, Hosts. This can be verified by visiting permissions tab of vcenter(and Datacenter, Datastore etc) and ensuring that either user or a group user belongs to has Read-Only role (at minimum) assigned to them.

Reply:
I added the `IMPORTANT` note above (lines 18-21) based on similar language from their docs. I did not include the verification step, however, because we should avoid documenting third-party UI functionality, which can change frequently. 

Does this now look good to you, @gnufied?